### ISSUE TITLE
proxy swallows error message when container fails to start

### ISSUE THREAD

**Issue by rade** [weaveworks/weave#1241]
````
$ docker run --rm -it gliderlabs/alpine
Error response from daemon: No command specified
$ weave launch
$ eval $(weave env)
$ docker run --rm -it gliderlabs/alpine
Error response from daemon: ba8ea356fa43646ec9e758af06d4443695483f27152c6ff40ca192ce188a5c01 container is not running.
$ docker logs weaveproxy
...
WARN: 2015/07/30 09:54:09.488245 Attaching container ba8ea356fa43646ec9e758af06d4443695483f27152c6ff40ca192ce188a5c01 to weave network failed: ba8ea356fa43646ec9e758af06d4443695483f27152c6ff40ca192ce188a5c01 container is not running.

WARN: 2015/07/30 09:54:09.488313 Error intercepting response: ba8ea356fa43646ec9e758af06d4443695483f27152c6ff40ca192ce188a5c01 container is not running.
````

**Comment by rade** [comment]
NB: I haven't tested this on 1.0, but I suspect the problem is present there too.

**Comment by rade** [comment]
> NB: I haven't tested this on 1.0, but I suspect the problem is present there too.

Indeed it is.

**Comment by paulbellamy** [comment]
Fixed in #1238

**Comment by paulbellamy** [comment]
The title of this bug was fixed in #1238.
The example provided was a different issue, and was fixed in #1253.
